agent/queryparser/helpers.go

```diff
@@ -201,7 +201,7 @@ func parseKeyValueFromComment(s string) (map[string]bool, error) {
 func prepareMultilineRegexp() error {
 	// to compile regexp only once
 	multilineOnce.Do(func() {
-		multilineRegexp, errMultiline = regexp.Compile(`(?s)\/\*(.*?)\*\/`)
+		multilineRegexp, errMultiline = regexp.Compile(`(?s)\/\*(.*?)\*\/`) //nolint:gocritic
```

Comment:
@JiriCtvrtka I believe we can use `regexp.MustCompile` everywhere in this file and simplify the code.

Reply:
and by using `MustCompile` we can migrate the initialization of `regexp`s to the top of the file